iocore/eventsystem/P_UnixEThread.h

```diff
@@ -79,7 +79,10 @@ TS_INLINE Event *
 EThread::schedule(Event *e)
 {
   e->ethread = this;
-  ink_assert(tt == REGULAR);
```

Comment:
This comes from the initial commit. Does anybody know why `schedule` has this assert?

Reply:
Yes. If the thread isn't `REGULAR` then it's a dedicated thread that only calls a single continuation - there is no event loop on which to schedule. See [here](https://github.com/apache/trafficserver/blob/master/iocore/eventsystem/UnixEThread.cc#L318).